src/components/ShortcutTooltip/ShortcutTooltip.tsx

```diff
@@ -1,35 +1,81 @@
 import * as React from 'react'
 import { Popup } from 'decentraland-ui'
-import { SimpleShortcut, ShortcutDefinition } from 'modules/keyboard/types'
-import { renderAlternative, renderCombination, renderSimple } from 'components/Modals/ShortcutsModal/ShortcutsModal'
+
+import { SimpleShortcut, ShortcutDefinition, ShortcutCombination, ShortcutAlternative } from 'modules/keyboard/types'
+import { mapLabel } from 'modules/keyboard/utils'
+
 import { Props } from './ShortcutTooltip.types'
 import './ShortcutTooltip.css'
 
+const renderCombination = (shortcut: ShortcutCombination) => {
+  let out: string[] = []
+
+  for (let i = 0; i < shortcut.value.length; i++) {
+    const label = mapLabel(shortcut.value[i])
+    out.push(label.length === 1 ? label.toUpperCase() : label)
```

Comment:
If you think it's worth it, we can re-use `renderSimple` here, something like:
```js
out.push(renderSimple({ type: 'simple', value: shortcut.value[i], title: shortcut.title }))
```

Same goes for [ShortcutModal](https://github.com/decentraland/builder/pull/202/files#diff-ac850bf1309f5f581c31c53143a70696R33)